face/capture/src/main/java/com/simprints/face/capture/screens/FaceCaptureViewModel.kt

```diff
@@ -73,11 +77,22 @@ internal class FaceCaptureViewModel @Inject constructor(
 
     fun initFaceBioSdk(activity: Activity) = viewModelScope.launch {
         val license = licenseRepository.getCachedLicense(Vendor.RANK_ONE)
-        if (!faceBioSdkInitializer.tryInitWithLicense(activity, license)) {
-            Simber.tag(CrashReportTag.LICENSE.name).i("License is invalid")
+        var licenseStatus = license.determineLicenseStatus()
+
+        if (licenseStatus == LicenseStatus.VALID) {
+            if (!faceBioSdkInitializer.tryInitWithLicense(activity, license!!.data)) {
```

Comment:
I'd advocate against using the `!!` operator unless absolutely necessary. Is it possible to avoid this?